packages/composables/src/useBreadcrumbs/useBreadcrumbs.ts

```diff
@@ -48,8 +66,38 @@ export function useBreadcrumbs(
     _breadcrumbs.value = [];
   };
 
+  const getCategoryBreadcrumbs = async (categoryId: string) => {
+    const response = await apiClient.invoke(
+      "readBreadcrumb get /breadcrumb/{id}",
```

Comment:
this endpoint is not only to load category breadcrumbs, I'm not sure about exposing it here, what is the designed flow here looks like?

Reply:
the endpoint is for both and takes more parameters
![image](https://github.com/user-attachments/assets/43c80ee8-cce7-4fed-963f-954f9eb3a5d4)


I just find `getCategoryBreadcrumbs` misleading as I would search for other types of breadcrumbs

Reply:
My point is - I would not keep this endpoint in the composable, it does not interfere with the logic and everyone can just use apiClient method for that and pass the data through `buildDynamicBreadcrumbs`

Reply:
done